### PR TITLE
[wip] Getting versioning and releases working

### PR DESCRIPTION
**Pull request by shana**
Based on #1282, getting CI to version and publish package artifacts

### CODE REVIEW

File: scripts/modules.ps1 [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+                Write-Host "``$Command`` failed" $output
+            } else {
+                Die $exitCode "``$Command`` failed" $output
+            }
+        }
+        $output
+    }
+
+    <#
+    function Run-Job([string[]]$Command, [switch]$Fatal, [switch]$Quiet, [int]$Timeout = 0) {
```
Comment (grokys) [inline scripts/modules.ps1]: Commented-out code.

File: scripts/modules/WiX.ps1 [hunk 2]
```diff
@@ -0,0 +1,43 @@
+Set-StrictMode -Version Latest
```
Comment (grokys) [inline scripts/modules/WiX.ps1]: Do we still need this now we're not building a WiX installer?

File: src/GitHub.InlineReviews/Peek/InlineCommentPeekResult.cs [hunk 3]
```diff
@@ -22,10 +22,11 @@ public InlineCommentPeekResult(InlineCommentPeekViewModel viewModel)
 
         public Action<IPeekResult, object, object> PostNavigationCallback => null;
 
-        public event EventHandler Disposed;
+        public event EventHandler Disposed = delegate { };
```
Comment (grokys) [inline src/GitHub.InlineReviews/Peek/InlineCommentPeekResult.cs]: What does the `= delegate { }` do here considering the `Disposed` event is being called?

File: src/GitHub.TeamFoundation.14/Connect/GitHubConnectSection.cs [hunk 4]
```diff
@@ -308,20 +308,20 @@ void HandleCreatedRepo(ILocalRepositoryModel newrepo)
         {
             Guard.ArgumentNotNull(newrepo, nameof(newrepo));
 
-            var msg = string.Format(CultureInfo.CurrentUICulture, Constants.Notification_RepoCreated, newrepo.Name, newrepo.CloneUrl);
-            msg += " " + string.Format(CultureInfo.CurrentUICulture, Constants.Notification_CreateNewProject, newrepo.LocalPath);
+            var msg = String.Format(CultureInfo.CurrentCulture, Constants.Notification_RepoCreated, newrepo.Name, newrepo.CloneUrl);
```
Comment (grokys) [inline src/GitHub.TeamFoundation.14/Connect/GitHubConnectSection.cs]: Why change `string` to `String` here?

File: src/GitHub.InlineReviews/Peek/InlineCommentPeekResult.cs (hunk 3 above)
Comment (jcansdale) [reply src/GitHub.InlineReviews/Peek/InlineCommentPeekResult.cs]: It stops CA complaining that `Disposed` isn't being used.

The alternatives are:
1. An attribute to ignore this error.
2. Explicitly defining `add { } and remove { }`.

This seemed like the least worst option. I'm open to alternatives.

File: src/GitHub.TeamFoundation.14/Connect/GitHubConnectSection.cs (hunk 4 above)
Comment (shana) [reply src/GitHub.TeamFoundation.14/Connect/GitHubConnectSection.cs]: I was copy-pasting the culture change and I guess I copy-pasted from somewhere that was uppercase

File: scripts/modules.ps1 (hunk 1 above)
Comment (shana) [reply scripts/modules.ps1]: Yes, I had a hard time getting this to work, powershell sucks

File: src/GitHub.InlineReviews/Peek/InlineCommentPeekResult.cs (hunk 3 above)
Comment (grokys) [reply src/GitHub.InlineReviews/Peek/InlineCommentPeekResult.cs]: What I meant was that on line 29 we're calling `Disposed?.Invoke(this, EventArgs.Empty);`, so `Disposed` _is_ being used. Will `Disposed` even work with the event defined like this?

**Pull request closed without merging**